advanced/Scripts/chronometer.sh

```diff
@@ -498,10 +498,6 @@ chronoFunc() {
         printFunc " RAM usage: " "$ram_perc%" "$ram_info"
         printFunc " HDD usage: " "$disk_perc" "$disk_info"
 
-        if [[ "$scr_lines" -gt 17 ]] && [[ "$chrono_width" != "small" ]]; then
-            printFunc "  LAN addr: " "${IPV4_ADDRESS/\/*/}" "$lan_info"
```

Comment:
I'm fine with the answer being "yes" here, by the way...

Is the `Lan addr` line being removed because it's simpler to not work out what it is in chronometer?

Reply:
And will this need changes to PADD as well?

Reply:
Eventually `IPV4_ADDRESS` will not be stored in `setupVars.conf` at all, so if PADD is sourcing from there...

Reply:
Yes. With automatically determined reply addresses depending on the interface a query arrives, displaying one single address will not be suitable in enough cases to be concerned. When changing IPv6 prefix makes a static address stored in setupVars rather useless, too.

Reply:
> And will this need changes to PADD as well?

![image](https://user-images.githubusercontent.com/1998970/124381729-f6a5fb00-dcbb-11eb-8f04-b6bd11b2bb16.png)

I don't know if we want to just remove the IP address from PADD or use some other method..?

Reply:
I guess we should just remove it

Otherwise, we could run something like
```
ip a | grep inet  | awk  '{print $2}'
```
which results in something like
```
127.0.0.1/8
::1/128
192.168.2.15/24
fd00::5858:711c:8625:34c4/64
fd00::8678:7fd8:8857:a6f9/64
2a02::3ab4:5555:2eb2:1368/64
2a02::a003:906e:9865:51cd/64
fe80::e7c6:5562:5687:8958/64
```
and we'd *just* have to pick the one the user wants to see. I bet this will be an issue most of the time.

Reply:
Have it display a random one from this list each time the variable is called. 😏

Reply:
Actually, we already have this in PADD:

![Screenshot from 2021-07-04 14-50-57](https://user-images.githubusercontent.com/16748619/124385765-61186480-dcd7-11eb-9b3a-d55b579842c5.png)

Reply:
See https://github.com/pi-hole/PADD/pull/161